kolibri/core/assets/src/vue/error-box/index.vue

```diff
@@ -21,9 +21,9 @@
     $trNameSpace: 'error',
     $trs: {
       errorHeader: 'Error',
-      errorButtonLabel: 'Hide Error',
+      errorButtonLabel: 'Hide error',
       // eslint-disable-next-line
-      explanation: `Sorry, we've encountered an issue. You may need to refresh the page or click one of the main navigation links.`,
+      explanation: `Unknown issue. You may need to refresh`,
```

Comment:
Fair enough. I was trying to follow the tone guide: [tone guidelines](https://material.io/guidelines/style/writing.html#writing-tone)

Not all users will be able to contact an administrator. Maybe something simple like this?

> Sorry, something went wrong. Please try refreshing the page